examples/demo/infrastructure_provisioning_with_i3s.rb

```diff
@@ -0,0 +1,81 @@
+# (c) Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+# This example is used to provision an Infrastructure with OS on Synergy with Image Streamer.
```

Comment:
Can you add the use case description here, just like you did in another file.

Reply:
@AnushaY1916 Corrected